.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+insert_final_newline = true
+
```

Comment:
I think this might make this more difficult to maintain, since if a file was added that requires different settings, it always requires changing this config, but it would likely not be noticed. Maybe it would be best to only explicitly mention which places should be formatted which way.

Reply:
I think that it's unlikely for a file to need different settings than those, but if you think it would make more sense to put them in the more specific section, I can.